src/Lykke.Job.CandlesHistoryWriter.Services/Settings/MigrationSettings.cs

```diff
@@ -5,7 +5,6 @@ namespace Lykke.Job.CandlesHistoryWriter.Services.Settings
 {
     public class MigrationSettings
     {
-        [Optional]
```

Comment:
Why it has been removed?

Reply:
Thepanis used while loading the job module not a vr  proper way. There is a cause to refactor it, but via separate LWDEV-8060. Just tried to eliminate ambiguity with minimal efforts here.

Reply:
Return it back, please. This change not affects code clarity, but since field is made required now, this change will complicate release, since we need to update settings of Spot and MT instances